src/components/Common/Inputs/Select.tsx

```diff
@@ -1,118 +0,0 @@
-import type { RefAttributes } from 'react'
```

Comment:
Same comment here as the others, maybe worth creating a follow up ticket to move the Select styles into the updated Select component along with a dedicated stylesheet